### PR TITLE
CLOUD-48896 smaller changes in the create cluster wizard

### PR DESCRIPTION
**Pull request by martonsereg**
@biharitomi 
- removed Consul server count even from advanced view
- moved aws dedicated instances checkbox to first page
- removed unnecessary panel from hdp repository config
- smaller buttons in wizard
- modified some text labels on buttons and some error messages

### CODE REVIEW

File: app/static/messages.properties [hunk 1]
```diff
@@ -300,9 +307,8 @@
     "cluster_form_hostgroup_label" : "Hostgroup configuration",
     "cluster_form_hostgroup_template_label" : "Template",
     "cluster_form_hostgroup_group_size_label" : "Group size",
-    "cluster_form_email_label" : "Email notification when cluster is provisioned",
-    "cluster_form_dedicated_label" : "Dedicated instances",
-    "cluster_form_ambari_repo_label" : "Ambari Repository config",
+    "cluster_form_email_label" : "Send email when cluster is ready",
```
Comment (biharitomi) [inline app/static/messages.properties]: We also send error emails too, so this label could be 'Send email notifications'.

**Comment by biharitomi** [comment]
I think the email notification parameter should not be an advanced option of a cluster (https://github.com/sequenceiq/uluwatu/pull/334/files#diff-34cd3975e969d54744995daf5b70febdL61).
Otherwise it LGTM.

**Comment by martonsereg** [comment]
Agreed that it's not an advanced option, but I think the email notification label can stay like this. On the UI we don't have to emphasize that a cluster creation can fail :)

**Comment by biharitomi** [comment]
Agreed if the cluster creation fails that label could mean the cluster is ready to be terminated :)

**Comment by biharitomi** [comment]
LGTM

**Pull request merged**